clean/lafayette_so.py

```diff
@@ -0,0 +1,106 @@
+import sys
+sys.path.append('../')
+import pandas as pd
+from lib.path import data_file_path, ensure_data_dir
+from lib.columns import clean_column_names, set_values
+from lib.clean import clean_dates, standardize_desc_cols
+from lib.uid import gen_uid
+
+
+def clean_names(df):
+    df.name = df.name.str.lower().str.strip().str.replace(',', '').fillna('')\
+        .str.replace('auzennejames', 'auzenne james', regex=False)\
+        .str.replace('st.cyrneil', 'st.cyr neil', regex=False)\
+        .str.replace('anceletjordan', 'ancelet jordan', regex=False)\
+        .str.replace('suarezrichard', 'suarez richard')\
+        .str.replace('file for records', '', regex=False)
+    parts = df.name.str.extract(r'(?:(\w+|\w+\.\w+) )?(?:([^ ]+) )?(.*)')
+    df.loc[:, 'last_name'] = parts[0]
+    df.loc[:, 'middle_name'] = parts[1]
+    df.loc[:, 'first_name'] = parts[2]
+    return df.drop(columns='name')
+
+
+def clean_complaint(df):
+    df.loc[:, 'charges'] = df.complaint.str.lower().str.strip()\
+        .str.replace(',', '', regex=False)\
+        .str.replace(r'(^file for records ?(only)?)', '', regex=True)\
+        .str.replace('unable to locate', '', regex=False)\
+        .str.replace('with drew complaint', '', regex=False)\
+        .str.replace(r'^courtesy/ identification$', 'courtesy/identification', regex=True)
+    return df.drop(columns='complaint')
+
+
+def clean_disposition(df):
+    df.disposition = df.disposition.str.lower().str.strip()\
+        .str.replace('non-sustained', 'unsustained', regex=False)\

```

Comment:
Is "unsustained" the same as "not sustained"? If it is then perhaps we want to standardize this?

Reply:
Changed to "not sustained" which should be the default.